config/config.yml.template

```diff
@@ -25,14 +28,15 @@ collections:
     summary: A collection of 1990s movies
   28 Days/Weeks Later:
     tmdb_id: 1565
-    system_name: 28 Days-Weeks Later
+    name_mapping: 28 Days-Weeks Later
   Anaconda:
     tmdb_id: 105995, 336560
 plex:
   library: Movies
   library_type: ### 'movie' or 'show' ###
   token: ###################
   url: http://192.168.1.5:32400
+  sync_mode: append
```

Comment:
Still debating whether this belongs under `plex` or outside, at the same level.

Reply:
my personal take is that it should be under plex because it seems odd as a standalone attribute and it really can't go under any other section

Reply:
Okay, fine with me.